frontend/libs/shared/ui-multiple-autocomplete/src/lib/multiple-autocomplete/multiple-autocomplete.component.ts

```diff
@@ -36,19 +38,22 @@ export class MultipleAutocompleteComponent implements OnInit, OnDestroy {
       .subscribe((data: string[]) => {
         this.selectedItemIds = data;
         this.control.markAsDirty();
-        this.input.nativeElement.blur();
         data.forEach((value) => this.added.emit(value));
       });
 
     this.selectedItemIds = this.control.value;
 
-    this.inputControl.valueChanges
-      .pipe(takeUntil(this.destroy$), startWith(null as string))
-      .subscribe((searchTearm) => {
-        if (typeof searchTearm === 'string') {
-          this._filter(searchTearm);
-        }
-      });
+    this.inputControl.valueChanges.pipe(takeUntil(this.destroy$), startWith(null as string)).subscribe((searchTerm) => {
+      if (typeof searchTerm === 'string') {
+        this._filter(searchTerm);
+      }
+    });
+  }
+
+  clearInput(): void {
+    // fixme: best to do this only via formControl ! ..... :(
```

Comment:
Was ist mit diesem Kommentar? Kann der raus?

Reply:
An der Stellet laesst sich der angezeigte Wert des Input Feldes in der GUI nicht ueber ein formControl.reset() zuruecksetzen. Desshalb wird in Zeile 55 + 56 beides seperat gesetzt. Das sollte aber eigentlich gehen.

Reply:
Nach Absprache: Kommentar bleibt drin, mit Task im Board.